target/linux/ramips/mt7621/base-files/etc/uci-defaults/05_fix-compat-version

```diff
@@ -0,0 +1,11 @@
+. /lib/functions.sh
+
+case "$(board_name)" in
```

Comment:
This file seems unnecessary. The config in `05_compat-version` should be enough.

Reply:
The config in `05_compat-version` only gets applied when using `sysupgrade -n` to discard settings. If user upgrades keeping settings without this, compat version will stay on 1.1 even though they have the new changes.

Reply:
But that won't happen because sysupgrade will surely brick their devices due to small kernel partition. We expect a new installation here, right? (Though user can manually restore their settings, then compat_version will roll back to 1.1 again, too bad.) If I'm wrong please correct me.

And I feel attaching a migration firmware to this PR or the wiki would be more useful than modifying the v23.05 branch. In this way user can find a specific file to update their older version, v21.x, v22.x and v23.x. We can ask user search wiki in the `DEVICE_COMPAT_MESSAGE`.

Reply:
> But that won't happen because sysupgrade will surely brick their devices due to small kernel partition.

Sysupgrade directly to a release with kernel more than 3MB will brick. That was the idea of having a transitional step through 23.x. Provided the new kernel is < 3MB (which it is in 23.x), then:

- First sysupgrade will update `ubnt.sh` and apply the new kernel layout from dts. Which is safe even if the kernel lands in kernel slot 2.
- The next subsequent update kernel will be installed to slot 1 (without the 3MB limitation.)
- User can keep settings through both these upgrades.


However I guess, as you mention, this could be done as a separate firmware build attached to wiki rather than changing 23.05. 

So i will drop this change from this PR.